JS Exercises/index.js

```diff
@@ -0,0 +1,110 @@
+// write to the console
+console.log("Hello from the JavaScript file")
+
+
+// variable
+let myName = "My name is Aron"
+
+console.log(myName)
+
+// camelCase my main city
+
+myMainCity = "I live in Berlin"
+
+// number
+let myFavNumber = "21"
+
+
+
+
+
+// String
+
+let myNewString = "Java is differnt than JavaScript"
+
+console.log(myNewString.length)
+
+console.log(myNewString.toUpperCase())
+
+console.log(myNewString.slice(0,12))
+
+console.log(myNewString.includes("JavaScript"))
+
+
+
+if(myFavNumber >20)(
+    console.log("21 is a positive number")
+                    )
+
+console.log(21%2)
+console.log(21%3)
+console.log(21%4)
+console.log(21%5)
+console.log(21%6)
+console.log(21%7)
+
+
+
+const numbers = [1, 2, 3];
+console.log(Math.max(...numbers)); // Output: 3
```

Comment:
Nice short variant & using Math is a good way avoiding unnecessary conditions. :)